content/ja/docs/concepts/workloads/pods/pod-overview.md

```diff
@@ -13,7 +13,7 @@ card:
 
 
 {{% capture body %}}
-## Podについて理解する
```

Comment:
It's probably OK to leave the original line. There's no requirement for the fragment identifiers to be in English.

Reply:
If it is left the original line, the link to here need to contains Japanese characters, which will encoded and hard to read in browser's address bar